### DISCUSSION TITLE
Flyby11 0.17

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/FlyOOBE#38] [Announcements]
- Optimized native bypass in several areas 
– Updated Fido script to download the latest Windows 11 builds https://github.com/builtbybel/Flyby11/issues/33 https://github.com/builtbybel/Flyby11/issues/31

## Bloatynosy Nue! The king is back... The original repo has been revived and is now online https://github.com/builtbybel/Bloatynosy

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Flyby11/releases/tag/0.17.170'>Flyby11 0.17</a>.</em>

**Comment by Kafka-1** [comment]
I actually, I use this is good. This tool to download woundus 11 on my tushiba. Laptop, but I have something to say. The newest version of it does not work well.I recommend to use the version before it

**Comment by vanminhpham6-dotcom** [comment]
duite d'après télécharger Windows 11 24h2 pro.ca ne fonctionne pas et lz messages qui dit des erreurs ?
